ts/sagas/startup.ts

```diff
@@ -208,8 +213,21 @@ function* initializeApplicationSaga(): IterableIterator<Effect> {
   // the wallet token is available,
   // proceed with starting the "watch wallet" saga
   const walletToken = maybeSessionInformation.value.walletToken;
-  yield fork(watchWalletSaga, sessionToken, walletToken, pagoPaApiUrlPrefix);
 
+  const isPagoPAQAEnabled: boolean = yield select<GlobalState>(
+    isPagoPAQAEnabledSelector
+  );
+
+  if (isPagoPAQAEnabled) {
```

Comment:
the check is reversed - and you can use the ternary operator here

```suggestion
yield fork(watchWalletSaga, sessionToken, walletToken, isPagoPAQAEnabled ? pagoPaApiUrlPrefixQa : pagoPaApiUrlPrefix);
```